src/main/java/io/github/dsse/helpers/SimpleECDSASigner.java

```diff
@@ -0,0 +1,30 @@
+package io.github.dsse.helpers;
+
+import io.github.dsse.models.Signer;
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+import java.security.PrivateKey;
+import java.security.Signature;
+import java.security.SignatureException;
+
+public class SimpleECDSASigner implements Signer {
```

Comment:
nit: Would it make sense to have comments for this class?  (and maybe the others too)

Reply:
Done.